.rubocop.yml

```diff
@@ -78,6 +78,10 @@ Lint/UnexpectedBlockArity:
 Lint/UnmodifiedReduceAccumulator:
   Enabled: true
 
+Lint/UnusedMethodArgument:
```

Comment:
why did you change this?

Reply:
to simplify this:
      def all_potential_upgrades(tile, tile_manifest: false) # rubocop:disable Lint/UnusedMethodArgument
to this:
       def all_potential_upgrades(tile, tile_manifest: false, selected_company: nil)

we have 4 not used reference inside the code like this, and the keyword parameter cannot be rewritten with a underscore prefix so rubicon complains about that.
If you think is better to avoid to have keywords parameters not used I can change all of them to normal parameters with default